taxonomy/models.py

```diff
@@ -0,0 +1,37 @@
+"""Models for lore taxonomy application"""
+
+from django.db import models
+
+from learningresources.models import (
+    Repository,
+    LearningResourceType,
+    LearningResource,
+)
+
+
+class Vocabulary(models.Model):
+    """Model for vocabulary table"""
+    repository = models.ForeignKey(Repository)
+    name = models.TextField()
+    description = models.TextField()
+    required = models.BooleanField()
+    vocabulary_type = models.CharField(max_length=1, choices=(
+        ("m", "managed"),
+        ("f", "free tagging")
+    ))
+    weight = models.IntegerField()
+
+    learning_resource_types = models.ManyToManyField(
+        LearningResourceType,
+        related_name="vocabularies"
+    )
+
```

Comment:
There should probably be a [unique_together](https://docs.djangoproject.com/en/1.8/ref/models/options/#unique-together) here so we don't get duplicate names within the same repository.